### PR TITLE
added terms of service for FoodHaus

### PR DESCRIPTION
**Pull request by XWangCS**

### CODE REVIEW

**Comment by tkevans** [comment]
Hey Chelsea @XWangCS can you rewrite the document into .txt format? I believe it's better than .rtf 
Besides, please don't let it have random characters like this one.

File: FoodHaus Terms of Service.rtf [hunk 1]
```diff
@@ -0,0 +1,301 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1504\cocoasubrtf830
```
Comment (tkevans) [inline FoodHaus Terms of Service.rtf]: what is this??

**Comment by HackBL** [comment]
I agree, as long as user sign up account, and they have to read the condition & terms part, they should understand our policy clearly. Don't make user confused the detailed, and the better way is make them to be .txt file first, then we'll probably create new VC to import them

**Comment by bojms45** [comment]
Also, if do this way drag the button into viewController, you cannot drag the button into other viewControllers in storeboard.

**Comment by XWangCS** [comment]
I just added it in Word format

**Comment by tkevans** [comment]
@XWangCS Please update any changes to "FoodHaus" repository and close this pull request WITHOUT merging.

**Pull request closed without merging**